### ISSUE TITLE
Flashing dvd subtitles with incorrect opacity

### ISSUE THREAD

**Issue by mjuhasz** [mjuhasz/BDSup2Sub#3]
Subtitles have incorrect length and they are not opaque after processing them with bdsup2sub. I have 2 dvds with 2 subtitles on each, and they all exhibit this behaviour.

**Comment by mjuhasz** [comment]
I learnt that a dvd subtitle frame can contain multiple control sequences. Each control sequence can have commands for changing the position, opacity, etc. of the subtitle. For each control sequence there is a delay value as well.
For further information see: http://dvd.sourceforge.net/spu_notes and http://dvd.sourceforge.net/dvdinfo/

BdSup2Sub is reading the subtitle attributes (alpha values, position, etc.) from the first control sequence only and ignores the rest. This is a limitation of the software and fixing it would be a feature to be added. I am planning to clean up the code before doing more than bugfixes.

The opacity will be correct if the subtitle is exported with a new palette (use this instead of "keep existing").
The flashing subtitle are caused by BdSup2Sup ignoring the delay values of the subsequent command sequences. I am planning to make a fix for this.

**Comment by mjuhasz** [comment]
At the moment I can fix the duration only. The opacity can be corrected by creating a new palette (and changing the alpha threshold if necessary). Special effects (fade-in, fade-out) will be lost after conversion however out of hundreds of dvds I only found 2 using them so this issue hardly ever happens in practice.

Fixed on master in commit https://github.com/mjuhasz/BDSup2Sub/commit/ccb70d56a55fd1e3194009d0d59022a013981084
